testing/resources/native/playbook.yml

```diff
@@ -0,0 +1,58 @@
+- hosts: couchbase
```

Comment:
I think this means we cover: https://issues.couchbase.com/browse/CMOS-77

Reply:
In the most technical sense yes - though it depends on where you draw the line between "kludge to support testing" "officially supported deployment mechanism"